src/app/containers/PageViewAnalytics/labelHelpers/index.test.js

```diff
@@ -204,17 +204,15 @@ describe('getBrowserViewPort', () => {
 });
 
 describe('getCurrentTime', () => {
-  const windowDate = window.Date;
-
-  afterEach(() => {
-    resetWindowValue('Date', windowDate);
-  });
-
   returnsNullWhenOffClient(getCurrentTime);
 
   it('should return hours, mins and seconds joined by "x"', () => {
-    const date = new Date(946729425000); // 12:23:45
-    setWindowValue('Date', () => date);
+    const mockDate = {
+      getHours: jest.fn().mockReturnValue('12'),
+      getMinutes: jest.fn().mockReturnValue('23'),
+      getSeconds: jest.fn().mockReturnValue('45'),
+    };
+    global.Date = jest.fn(() => mockDate);
```

Comment:
Can you reset this after please?

Reply:
Yeah can do, FYI it does reset after the test file

Reply:
done